build/images/ovs/build.sh

```diff
@@ -163,6 +168,10 @@ elif [ "$DISTRO" == "ubi" ]; then
 elif [ "$DISTRO" == "windows" ]; then
     image="antrea/windows-ovs"
     build_args="--build-arg OVS_VERSION=$OVS_VERSION"
+    if [[ ${DOCKER_REGISTRY} != "" ]]; then
+        image="${DOCKER_REGISTRY}/antrea/windows-ovs"
+        build_args+=" --build-arg DOCKER_REGISTRY=$DOCKER_REGISTRY"
```

Comment:
these build args will be consumed by `build/images/ovs/Dockerfile.windows`, but I don't see `DOCKER_REGISTRY` as an `ARG` in that Dockerfile.

The `FROM` directive reads like this:
```
FROM --platform=linux/amd64 ubuntu:24.04 AS antrea-windows-builder
```
So it will be suing `ubuntu:24.04` unconditionally, which is ok because of the `docker tag` command (line 139).

Reply:
Yes it should be removed, thanks!